main.py

```diff
@@ -1773,10 +1775,28 @@ def start_client(window: sublime.Window, config: ClientConfig):
         window.status_message("Starting " + config.name + "...")
     debug("starting in", project_path)
 
+    # Create a dictionary of Sublime Text variables
     variables = window.extract_variables()
-    expanded_args = list(sublime.expand_variables(os.path.expanduser(arg), variables) for arg in config.binary_args)
+    variables["sublime_path"] = os.path.dirname(sublime.executable_path())
+    variables["sublime_libs"] = ';'.join(sys.path)
```

Comment:
Are these variables of use to other LSP users who are not Sublime Text package developers?
If not, please remove them. I requested this in the issue:

> If this means hardcoding the path to sublime.executable_path() in the project file that's probably ok?

Reply:
Those variable enable the usage of `${sublime_path}` and `${sublime_libs}` variables in the settings file next to `${project}` etc. such as in the build-system. Adding them to `variables` does neither harm nor does it take any time. The whole process of reading/expanding the environment variables takes about 0.2ms upon server startup. The `${sublime_path}` may help people using portable setups with files located relative to the sublime text executable, while `${sublime_libs}` may be most useful for package authors. But as it contains a variable set of library paths including dependencies it is not so easy to replace by hard coded paths - makes life just much easier.

These are my arguments to keep it.

Reply:
I appreciate your arguments, but they confirm my belief that it is an edge case.

The whole `env` feature is a convenience only you have requested so far. Once it has accumulated some users we can always add in functionality. Removing functionality is *much* harder.

Reply:
> The whole env feature is a convenience only you have requested so far ...

Not to forget guys like the one who requested https://github.com/palantir/python-language-server/issues/143 😄 

Most people are just happy to consume. 

Is keeping the exe path for portable users ok?

Reply:
> @tomv564 The whole env feature is a convenience only you have requested so far.

Me too.